packages/dashboards/src/lib/configurator/components/widgets/table/columns-editor-v2/table-columns-configuration-v2.component.html

```diff
@@ -68,7 +68,7 @@
         </div>
     </div>
     <nui-message class="m-4 d-block"
-                 *ngIf="dataSourceManager?.dataSource && !(dataSourceManager?.dataSource?.busy | async) && !dataSourceFields.length"
+                 *ngIf="dataSourceManager?.dataSource && (dataSourceManager?.dataSource?.busy | async) === false && !dataSourceFields.length"
```

Comment:
I wonder why this is no longer allowed. Kind of unfortunate. Not a big deal.

Reply:
Because of an error in eslint "Async pipes should not be negated. Use (observable | async) === (false | null | undefined) to check its value instead  @angular-eslint/template/no-negated-async". 
We can turn off this rule if the first variant is better for us.

Reply:
Meh, it's fine to keep it I guess.

Reply:
this is actually a pretty weird rule. Lets disable it and discuss it's rational with the Alpha team .